### PR TITLE
Done Task1. Waiting for checking.

### PR DESCRIPTION
**Pull request by SmooZzzie**
-----------
P.S. Idk why, but in my tests @Before and @BeforeAll annotations didn't work. Libs were added, but nothing happened.

### CODE REVIEW

File: ArtemShchepets/src/week1/tests/testBill.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+package week1.tests;
+
+import org.junit.Assert;
+import org.testng.annotations.Test;
+import week1.data.Bill;
+import week1.data.Product;
+import week1.data.Seller;
+import week1.data.Time;
+
+public class testBill {
```
Comment (presly808) [inline ArtemShchepets/src/week1/tests/testBill.java]: remember about naming

File: ArtemShchepets/src/week1/tests/testBill.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                + testProduct2.showInfo() + "\n"
+                + testProduct3.showInfo() + "\n"
+                + testProduct4.showInfo() + "\n"
+                + "12:33:50\n" + "Seller: Valya, age: 22" + "\n***BILL IS OPENED***";
+
+        String actual = testBill.showInfo();
+
+        boolean isSame = expected.equals(actual);
+
+        Assert.assertTrue(isSame);
```
Comment (presly808) [inline ArtemShchepets/src/week1/tests/testBill.java]: use assertEquals for comparing

File: ArtemShchepets/src/week1/tests/testBill.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    @Test
+    public void testCloseBill() {
```
Comment (presly808) [inline ArtemShchepets/src/week1/tests/testBill.java]: you may remove expected and actual variables

File: ArtemShchepets/src/week1/tests/testProduct.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    Product testProduct1 = new Product("Milk", 11.20, "#03242341");
+    Product testProduct2 = new Product("Cheese", 2.05, "#0341");
+    Product testProduct3 = new Product("Water", 33.5, "#01");
+    Product testProduct4 = new Product("Cake", 7.55, "#222");
+
+    @Test
+    public void testShowInfo() {
+        String expected = "Product \"Milk\", price: 11,20, barcode: #03242341";
+        String actual = testProduct1.showInfo();
+        Assert.assertEquals(actual, expected);
```
Comment (presly808) [inline ArtemShchepets/src/week1/tests/testProduct.java]: I would use contains instead of equals

File: ArtemShchepets/src/week1/data/Bill.java [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+        String resultString = "***BILL***\n";
+
+        for (int i = 0; i < billList.length; i++) {
+            resultString += billList[i].showInfo() + "\n";
+        }
+
+        resultString += time.toString() + "\n" + seller.toString();
+
+        return   (isClosed) ? resultString + "\n***BILL IS CLOSED***" : resultString + "\n***BILL IS OPENED***";
```
Comment (presly808) [inline ArtemShchepets/src/week1/data/Bill.java]: (isClosed) doesn't need braces

File: ArtemShchepets/src/week1/data/Bill.java [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+                productListSize++;
+            }
+        } else System.out.println("Sorry, bill is closed!");
+    }
+
+    public String showInfo() {
+
+        String resultString = "***BILL***\n";
+
+        for (int i = 0; i < billList.length; i++) {
```
Comment (presly808) [inline ArtemShchepets/src/week1/data/Bill.java]: what would happen if your billList contained some null elements?

**Review by presly808 — COMMENTED** [review]
In general you did well

File: ArtemShchepets/src/week1/data/Bill.java (hunk 6 above)
Comment (SmooZzzie) [reply ArtemShchepets/src/week1/data/Bill.java]: https://github.com/presly808/aco22/pull/11/commits/39ad6288fa4d4b6c4408a80ad7e46ead92db86b5 - fixed

File: ArtemShchepets/src/week1/data/Bill.java (hunk 5 above)
Comment (SmooZzzie) [reply ArtemShchepets/src/week1/data/Bill.java]: https://github.com/presly808/aco22/pull/11/commits/39ad6288fa4d4b6c4408a80ad7e46ead92db86b5 - fixed

File: ArtemShchepets/src/week1/tests/testBill.java (hunk 1 above)
Comment (SmooZzzie) [reply ArtemShchepets/src/week1/tests/testBill.java]: https://github.com/presly808/aco22/pull/11/commits/c952f7ab76ee2e83dd5e84b1033681dee07222b6 - fixed

File: ArtemShchepets/src/week1/tests/testBill.java (hunk 2 above)
Comment (SmooZzzie) [reply ArtemShchepets/src/week1/tests/testBill.java]: https://github.com/presly808/aco22/pull/11/commits/7771fe3f9908d06846f900b9ba28a78b95242329 - fixed

File: ArtemShchepets/src/week1/tests/testBill.java (hunk 3 above)
Comment (SmooZzzie) [reply ArtemShchepets/src/week1/tests/testBill.java]: https://github.com/presly808/aco22/pull/11/commits/8cc7e9b1f5cb4f1e5443a47c26e9cc20acc8e872 - fixed

File: ArtemShchepets/src/week1/tests/testProduct.java (hunk 4 above)
Comment (SmooZzzie) [reply ArtemShchepets/src/week1/tests/testProduct.java]: Can't find such method in class Assert.

**Pull request merged**